src/styles/_components/_animation.scss

```diff
@@ -48,6 +48,15 @@ $transition: $animation-duration $cubic-bezier;
     }
 }
 
+@keyframes loading {
+    0%{
+        background-position: -468px 0
```

Comment:
This does not depend on the element size, this is just to give animation a "delay" (see here: http://cloudcannon.com/deconstructions/2014/11/15/facebook-content-placeholder-deconstruction.html).

Do you think we should put it into a variable?